import-dashboards.py

```diff
@@ -242,6 +244,47 @@ def copy_apps():
             shutil.copytree(source_dir, dest_dir)
 
 
+def create_folders():
+    for folder in SET_OF_TAGS.keys():
+        path = PMM_PLUGIN_DIR + folder
+        try:
+            os.mkdir(path)
+        except OSError:
+            print ' * Creation of subfolder %s failed' % path
+        else:
+            print ' * Successfully created subfolder %s' % path
+
+
+def move_dashboards_into_folders():
+    for filename in os.listdir(PMM_PLUGIN_DIR):
+        if os.path.isfile(os.path.join(PMM_PLUGIN_DIR, filename)):
+            with open(PMM_PLUGIN_DIR + filename, 'r') as dashboard_file:
```

Comment:
I think it's better to do `os.path.join(PMM_PLUGIN_DIR, filename)` ones abowbe `if`

Reply:
Added